frontend/pages/playscreen.vue

```diff
@@ -305,26 +316,26 @@ const checkAnswer = (direction) => {
 
 const submitTestConfigResult = async () => {
   try {
-    const token = localStorage.getItem('token'); // Get token from localStorage
-    const testConfigId = getTestConfigId(); // Get test_config_id based on difficulty and mode
+    const token = localStorage.getItem('token'); // Erhalte Token aus localStorage
+    const testConfigId = getTestConfigId(); // Erhalte test_config_id basierend auf Schwierigkeitsgrad und Modus
 
     const payload = {
-      test_config_id: testConfigId, // Use the dynamically retrieved test_config_id
+      test_config_id: testConfigId, // Verwende die dynamisch abgerufene test_config_id
       time: new Date().toISOString(),
       correct_answers: score.value,
-      wrong_answers: randomizedItems.value.length - score.value,
-      item_config_result_ids: itemConfigResultIds.value, // Use the stored result IDs from item_config submissions
+      wrong_answers: detailedResults.value.length - score.value, // Korrigiert hier
+      item_config_result_ids: itemConfigResultIds.value, // Verwende die gespeicherten Ergebnis-IDs von item_config Submissionen
     };
 
     await $fetch(`${config.public.backendUrl}/api/test_config_results/`, {
       method: "POST",
       headers: {
-        Authorization: `Bearer ${token}`, // Add Bearer token to the request
+        Authorization: `Bearer ${token}`, // Füge Bearer Token zur Anfrage hinzu
       },
       body: payload,
     });
   } catch (error) {
-    // Show error toast
+    // Zeige Fehler-Toast
     toast.add({
       title: 'Error',
       description: 'Failed to submit test results. Please try again.',
```

Comment:
I think we should provide error messages in german as well, do you agree?

Reply:
is translated